cmd/packaging/windows-msi.go

```diff
@@ -129,34 +203,41 @@ func windowsMsiProcessFiles(path string) {
 
 	for _, f := range files {
 		p := filepath.Join(path, f.Name())
-		relativePath := strings.Split(strings.Split(p, "flutter_assets"+string(filepath.Separator))[1], string(filepath.Separator))
+		relativePath := strings.Split(strings.Split(p, "flutter_assets"+fileSeparator)[1], fileSeparator)
 		if f.IsDir() {
 			directoriesFileContent = append(directoriesFileContent,
-				`<Directory Id="FLUTTERASSETSDIRECTORY_`+strings.Join(relativePath, "_")+`" Name="`+f.Name()+`">`,
+				fmt.Sprintf("<Directory Id=\"FLUTTERASSETSDIRECTORY_%s\" Name=\"%s\">", hashSha1(strings.ReplaceAll(strings.ReplaceAll(strings.Join(relativePath, "_"), " ", "_"), "-", "_")), f.Name()),
 			)
-			windowsMsiProcessFiles(p)
+			windowsMsiProcessFiles(p, fileSeparator)
 			directoriesFileContent = append(directoriesFileContent,
-				`</Directory>`,
+				"</Directory>",
 			)
 		} else {
 			if len(relativePath) > 1 {
 				directoryRefsFileContent = append(directoryRefsFileContent,
-					`<DirectoryRef Id="FLUTTERASSETSDIRECTORY_`+strings.Join(relativePath[:len(relativePath)-1], "_")+`">`,
+					fmt.Sprintf("<DirectoryRef Id=\"FLUTTERASSETSDIRECTORY_%s\">", hashSha1(strings.ReplaceAll(strings.ReplaceAll(strings.Join(relativePath[:len(relativePath)-1], "_"), " ", "_"), "-", "_"))),
```

Comment:
Instead of escaping the double-quotes, this could be a raw string literal (using backticks). That would greatly improve the readability of this (and other) lines. I see it was using backticks already, wondering why those were replaced with double quotes?